manifests/gateways/istio-egress/templates/deployment.yaml

```diff
@@ -56,7 +56,9 @@ spec:
             - -c
             - sysctl -w kernel.core_pattern=/var/lib/istio/core.proxy && ulimit -c unlimited
           securityContext:
-            privileged: true
```

Comment:
this is the core dump, I feel like sysctl may need root? maybe not though, but it's off by default

Reply:
you're right, these are wrong. will change back